packages/cli/src/utils/solidity.ts

```diff
@@ -35,15 +36,9 @@ export function compilerSettingsMatch(s1: CompilerVersionOptions, s2: CompilerVe
 }
 
 export function getImports(source: string): string[] {
-  // Copied from https://github.com/nomiclabs/buidler/blob/1cd52f91d7f8b6756c5ac33b78f93b151b072ea4/packages/buidler-core/src/internal/solidity/imports.ts
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const parser = require('solidity-parser-antlr');
-  const ast = parser.parse(source, { tolerant: true });
-
-  const importedFiles: string[] = [];
-  parser.visit(ast, {
-    ImportDirective: (node: { path: string }) => importedFiles.push(node.path),
-  });
-
-  return importedFiles;
+  try {
+    return getMetadata(source).imports;
+  } catch {
+    return [];
```

Comment:
I'd output a warning at least, just in case that the error is different

Reply:
Wouldn't that be a bug in the parser? It shouldn't crash when fed valid Solidity source